### PR TITLE
feat: flow stats filters

### PR DESCRIPTION
**Pull request by csiyang**
# Description

### Issue

Add custom filter for editor stats

[Link to Basecamp Todo](https://3.basecamp.com/3105655/projects)

### Solution

- Add local provider for filter values
- Add filter component
- Update stats query filter

# External Changes

n/a

# Additional information

n/a

### CODE REVIEW

File: apps/journeys-admin/src/components/PlausibleFilter/RangePicker/RangePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'right'
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'right'
+      }}
+      sx={{
+        color: 'pink',
```
Comment (levi-zustiak) [inline apps/journeys-admin/src/components/PlausibleFilter/RangePicker/RangePicker.tsx:43]:
```suggestion
```

File: apps/journeys-admin/src/components/PlausibleFilter/RangePicker/RangePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        '.rdp': {
+          '--rdp-background-color': (theme) => theme.palette.primary.main,
+          '--rdp-accent-color': (theme) => theme.palette.primary.main,
+          '.rdp-button': {
+            ':hover': {
```
Comment (levi-zustiak) [inline apps/journeys-admin/src/components/PlausibleFilter/RangePicker/RangePicker.tsx:48]: Prevents the number from "disappearing" on hover for disabled dates

```suggestion
            ':hover:enabled': {
```

**Review by levi-zustiak — CHANGES_REQUESTED** [review]
I didn't see the filter added in JourneyFlow. Will it be added in a separate PR?

**Comment by csiyang** [comment]
Closing until the feature is needed

**Pull request closed without merging**